src/services/native-mobile-interface/notifications.ts

```diff
@@ -18,3 +18,9 @@ export class ResetNotificationsBadgeCount extends NativeMobileMessage {
     super(MessageType.RESET_NOTIFICATIONS_BADGE_COUNT, {})
   }
 }
+
+export class PromptPushNotificationPermissions extends NativeMobileMessage {
+  constructor() {
+    super(MessageType.PROMPT_PUSH_NOTIFICATION_REMINDER, {})
```

Comment:
We should make the second param in NativeMobileMessage default to empty object so we don't have to pass that in.

Reply:
doing it